src/apps/chifra/internal/blocks/handle_uniq.go

```diff
@@ -17,6 +18,8 @@ import (
 )
 
 func (opts *BlocksOptions) HandleUniq() (err error) {
+	readOnly := true
```

Comment:
I think, since we're reading blocks, we can let the user decide and use `opts.Cache`. You may have to remove the restriction in `Validate`. The restriction was only there because the C++ code was confusing.

Add three test cases to `getBlocks.csv`

```
chifra blocks --decache number
chifra blocks --cache --uniq 7001001
chifra blocks --uniq 7001001
```

The second and third should get identical results.